lib/minidump.js

```diff
@@ -36,6 +36,27 @@ function execute (command, args, callback) {
 var globalSymbolPaths = []
 module.exports.addSymbolPath = Array.prototype.push.bind(globalSymbolPaths)
 
+module.exports.moduleList = function (minidump, callback) {
+  fs.readFile(minidump, (err, data) => {
+    if (err) return callback(err)
+    const { streams } = require('./format').readMinidump(data)
+    const moduleList = streams.find(s => s.type === 4)
```

Comment:
Since the `4` hardcoding happens twice -- here and as a key in the `streamTypes` object -- might be better to make this an actual symbolic constant, e.g. 

```js
const MD_MODULE_LIST_STREAM = 4;
...
const streamTypes = {
￼   MD_MODULE_LIST_STREAM: (stream, buf) => {
...
const moduleList = streams.find(s => s.type === MD_MODULE_LIST_STREAM)
```